modules/@apostrophecms/storybook/template/public/lib/mock-images.js

```diff
@@ -71,7 +71,9 @@
     ]
   };
   window.apos.http.getResponses['/api/v1/@apostrophecms/image'] = {
-    results: createImages(30)
+    results: createImages(31),
```

Comment:
Actually related to this.

Reply:
why not make this higher than the default or knob'able to toggle experiences

Reply:
I'm not sure what you mean. If it was higher it wouldn't fit the grid.